PluginGenerator/Resources/Rules/PluginRulesDefinition.java

```diff
@@ -24,11 +24,11 @@ public PluginRulesDefinition() {
   }
 
   protected String rulesDefinitionFilePath() {
-    return "/resources/rules.xml";
+    return "/resources/[RESOURCE_ID].rules.xml";
   }
 
   protected String sqaleDefinitionFilePath() {
-    return "/resources/sqale.xml";
+    return "/resources/[RESOURCE_ID].sqale.xml";
```

Comment:
Miscellaneous: changed the resource names to be unique. Strictly speaking this shouldn't be necessary since SonarQube loads each plugin in isolation so it should not be possible to have multiple resources with the same name. However, there's no harm in the names being unique and it removes one possible source of confusion that I hit during testing (multiple generated jars on the class path, so the wrong resource file was being picked up).